integrations/client_server_name.js

```diff
@@ -1,10 +1,7 @@
-import * as AdobeAnalytics from "./AdobeAnalytics/constants";
-
 // from client native integration name to server identified display name
 // add a mapping from Rudder identified key names to Rudder server recognizable names
 const clientToServerNames = {
   All: "All",
-  [AdobeAnalytics.NAME]: AdobeAnalytics.DISPLAY_NAME,
```

Comment:
Add entry for adobe

Reply:
It's probably not required. I'll fix all these issues when I reintroduce these changes.